zeppelin-zengine/src/test/java/org/apache/zeppelin/notebook/NotebookTest.java

```diff
@@ -150,13 +150,14 @@ public void testSchedule() throws InterruptedException, IOException{
 		note.setConfig(config);
 		notebook.refreshCron(note.id());
 		Thread.sleep(1*1000);
-		dateFinished = p.getDateFinished();
-		assertNotNull(dateFinished);
-
+		
 		// remove cron scheduler.
 		config.put("cron", null);
 		note.setConfig(config);
 		notebook.refreshCron(note.id());
+		Thread.sleep(1000);
+		dateFinished = p.getDateFinished();
+		assertNotNull(dateFinished);
 		Thread.sleep(1*1000);
 		assertEquals(dateFinished, p.getDateFinished());
```

Comment:
Okay, got it. Thanks for explanation!

Merging it if there're no more discussions.